### PR TITLE
fix for game_model not loading lottoprice

### PR DESCRIPTION
**Pull request by joshgit**

### CODE REVIEW

File: py3/game.py [hunk 1]
```diff
@@ -1151,39 +1167,44 @@ def lotto():
...
         elif prob < 0.896875:
-            money += lottoprice * 1000
+            money += lp * 1000
         elif prob < 0.897:
-            money += lottoprice * 20000
+            money += lp * 20000
         else:
             money /= 2.0
-        lottoprice.set('Lotto ($' + str(round(game_state.lottoprice, 1)) + ')')
+        lottoprice.set('Lotto ($' + str(round(lp, 1)) + ')')
```
Comment (qlstudio) [inline py3/game.py]: @joshgit , you forgot to set the lotto price value in the game_state instance

Comment (joshgit) [inline py3/game.py]: @qlstudio what do you mean?  stuff was pretty broken when i did a fresh clone...

**Comment by DerpfacePython** [comment]
Wait wait wait, why are you removing the -illions? They were required so that the money, when loaded from the savefile, wouldn't reach an `OverflowError` so quickly.

**Comment by DerpfacePython** [comment]
No, but when the games to savefile, the number for money gets carried over. Also, because we're setting money to int, the money will get large enough to `Overflow`. You should probably change the thing to `long` as well.

Also, the class that you made for Py2 for `game_model`had all these getter and setter functions (e.g. `get_printmoney`) that weren't really necessary, so we got rid of them.

File: py3/game.py (hunk 1 above)
Comment (qlstudio) [inline py3/game.py]: shouldn't you have `game_model.lottoprice = lottoprice`?

**Comment by qlstudio** [comment]
> I'm not really sure what goes on, but it wouldn't convert through savefile for Python 2

What do you mean? Python 3 uses byte a85-base encoding, but Python 2 uses string hexadecimal encoding...

**Comment by qlstudio** [comment]
> You should probably change the thing to long as well

The `long` data type got removed in python 3. Proof:

```python
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  6 2015, 01:38:48) [MSC v.1900 32 bit (Intel)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> 63438726487326587465876328764873264873265482376463258732465874365874365837465873276487326
63438726487326587465876328764873264873265482376463258732465874365874365837465873276487326
>>> long
Traceback (most recent call last):
  File "<pyshell#1>", line 1, in <module>
    long
NameError: name 'long' is not defined
>>> 
```

File: py3/game.py (hunk 1 above)
Comment (qlstudio) [inline py3/game.py]: shouldn't you have `game_model.lottoprice = lottoprice`?

Comment (joshgit) [inline py3/game.py]: Why?  Do the lotto results affect future lotto price?

**Comment by joshgit** [comment]
It's still transparently handling integer numbers that are too large to fit into an int.
$python3
Python 3.5.1+ (default, Feb 24 2016, 10:41:47) 
[GCC 5.3.1 20160222] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 2 ** 32
4294967296
>>> 2 ** 64
18446744073709551616
>>> 2 ** 128
340282366920938463463374607431768211456
>>>

**Comment by DerpfacePython** [comment]
Oh.

Well, by all means, go ahead.

**Pull request merged**